### PR TITLE
Updated changelogs with CVEs / security fixes

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

File: 10.Mender-server/docs.md [hunk 1]
```diff
@@ -4135,6 +4141,16 @@ New changes in integration since 3.3.0:
 
 _Released 03.10.2023_
 
+### Security fixes
```
Comment (lluiscampos) [inline 10.Mender-server/docs.md:4144]: Having a separate section here looks very good. Nice initiative.

Could you please extend it also to 3.7.4 ([example](https://github.com/mendersoftware/mender-docs-changelog/blob/2079d62fdd53fc1687d717b97e67f9e4c40f2756/10.Mender-server/docs.md#bug-fixes-3)) and 7.6.4 changelogs?

File: 10.Mender-server/docs.md [hunk 2]
```diff
@@ -2827,6 +2827,12 @@ New changes in workflows-enterprise since 2.3.0:
 
 _Released 09.25.2022_
 
+### Security fixes
+
+* Fixed CVE-2022-41324 - Read-only API access can leak secrets in Mender.
+  See the official announcement for more information:
+  https://mender.io/blog/cve-2022-45929-cve-2022-41324-improper-access-control-for-low-privileged-users
```
Comment (lluiscampos) [inline 10.Mender-server/docs.md:2834]: This fix was not released in this version. Please remove

Comment (olehermanse) [reply 10.Mender-server/docs.md:2834]: Okay, please see comments in ticket: https://northerntech.atlassian.net/browse/SEC-591

File: 10.Mender-server/docs.md (hunk 1 above)
Comment (olehermanse) [reply 10.Mender-server/docs.md]: Done, updated 3.6.4 and 3.7.4 in the latest commit.

**Review by lluiscampos — CHANGES_REQUESTED** [review]

File: 10.Mender-server/docs.md [hunk 3]
```diff
@@ -77,19 +83,6 @@ New changes in integration since 3.7.3:
...
-New changes in useradm-enterprise since 1.22.0:
-
-##### Bug Fixes
-
-* Fixed CVE-2024-37019. See the official announcement for more information:
-  https://mender.io/blog/cve-2024-37019-account-takeover-using-saml
-
-  When assigning SSO configuration to the user, check if the SSO with
-  given Id is configured for the tenant the user has access to.
-  ([SEC-1106](https://northerntech.atlassian.net/browse/SEC-1106))
```
[the remark this answers is not in the document]
Comment (olehermanse) [reply 10.Mender-server/docs.md:91]: ACK, amended.

**Review by lluiscampos — APPROVED** [review]
Thank you :+1:

**Pull request merged**